deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -260,6 +259,7 @@ class ChatViewController: UITableViewController {
         if RelayHelper.sharedInstance.isForwarding() {
             askToForwardMessage()
         }
+        prepareContextMenu()
```

Comment:
how is this change related to the gallery context menu? or what does that fix?

Reply:
`UIMenuController` is a singleton (by Apple). I change it's options in Gallery, so I have to reset these options when moving back from Gallery.

Reply:
thanks for explanation :)